ScoreSaber/Plugin.cs

```diff
@@ -64,24 +61,14 @@ public Plugin(IPALogger logger, PluginMetadata metadata, Zenjector zenjector) {
             zenjector.Install<RecordInstaller, StandardGameplayInstaller>();
             zenjector.Install<RecordInstaller, MultiplayerLocalActivePlayerInstaller>();
             zenjector.UseAutoBinder();
-
-            LibVersion = Assembly.GetExecutingAssembly().GetName().Version;
-            HttpInstance = new Http(new HttpOptions() { baseURL = "https://scoresaber.com/api", applicationName = "ScoreSaber-PC", version = LibVersion });
+            harmony = new Harmony("com.umbranox.BeatSaber.ScoreSaber");
+            harmony.PatchAll(Assembly.GetExecutingAssembly());
```

Comment:
Considering we are no longer honoring `Settings.disableScoreSaber`, we should at the very least add comment in the Settings.cs file that this setting no longer does anything.

Reply:
fixed